changelogs/unreleased/gh-7486-prohibit-roll-back-due-to-replication_synchro_timeout.md

```diff
@@ -0,0 +1,16 @@
+## feature/replication
+
+This patch is designed to prohibit automatic transaction rollback due to
```

Comment:
Let's please not mention the patch in the changelog.
When creating release changelogs, we use a script to paste the changelog files as is (see [here](https://github.com/tarantool/tarantool/releases/tag/3.2.0), for example).

So the two bullet points below are enough.